cmd/launch.go

```diff
@@ -67,6 +67,8 @@ func init() {
 		"The absolute filepath to a bash script passed to the instance and executed after the instance starts (user data)")
 	launchCmd.Flags().StringToStringVar(&flagConfig.UserTags, "tags", nil,
 		"The tags applied to instances and volumes at launch (Example: tag1=val1,tag2=val2)")
+	launchCmd.Flags().StringVar(&flagConfig.CapacityType, "capacity-type", "",
+		fmt.Sprintf("Launch instance as \"%s\" or \"%s\"", question.DefaultCapacityTypeText.OnDemand, question.DefaultCapacityTypeText.Spot))
```

Comment:
I'm guessing we're avoiding specifying a default because even though we technically have one in the [simpleConfig (On-Demand)](https://github.com/awslabs/aws-simple-ec2-cli/blob/20ee19612a5a2878bceea2e41fa86af440ef1e3f/pkg/ec2helper/ec2helper.go#L1068), it will be ignored once the default config is replaced by user config. Should this default (and the ability to save and use a user-defined config) be documented?

Reply:
If we specify a default in the third parameter of `Flags().StringVar()` but the user doesn't use that flag when invoking, the flag default will always override the value in their configuration file, which is not the correct behavior. By leaving the default here empty, we ensure that we preserve their preferences until they explicitly use the flag.

But I could reword the help text to say "Launch instance as On-Demand (the default) or Spot". That way we still have the correct behavior in the code, but the user will know which one is the default if they haven't specified anything else (either in the flags or in their config file).

Would that be sufficient, or would you also like to see an explanation somewhere else?

Reply:
I think that would be sufficient, thanks 👍

Reply:
Fixed in 92e9c465e9fac8348b408e0b70342eff9f813943